### PR TITLE
Update PowerShell SDK Version and Correct mistakes in the README file

### LINKED ISSUE #4369: Update PowerShell SDK Version

### PR DESCRIPTION
**Pull request by skanda890**
- [X] I have signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs).
- [X] This pull request is related to an issue.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-cli/pull/4368)

This closes #4369

### CODE REVIEW

**Comment by skanda890** [comment]
I think this PR was created by accident.

**Comment by yao-msft** [comment]
> I think this PR was created by accident.

Ok, thanks for letting us know. I'll close.

**Comment by skanda890** [comment]
This closes #4369

**Comment by skanda890** [comment]
Huh? Can you simplify.

**Comment by skanda890** [comment]
Merge this.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-cli/blob/91ee267ccd1bb6d9f976153e195572a5ee9746d8/src/Microsoft.Management.Configuration.Processor/Microsoft.Management.Configuration.Processor.csproj#L42

**Comment by skanda890** [comment]
> Ah ok, I thought the change was for tests projects only so we do not quite care. Then I saw we have one reference in Microsoft.Management.Configuration.Processor.csproj. Please update that one too so we can s what Stnk toldget the change in.
> 
> 

The changes are made by Synk.

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
...
 
-If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
+If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "Winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
 
-> **Note**: Group policy may be configured and modify configured sources. Run `winget --info` to see any configured policies.
+> **Note**: Group policy may be configured and configured sources. Run `winget --info` to see any configured policies.
```
Comment (Trenly) [inline README.md:10]: This sentence doesn't make sense with your change. The original has proper grammar

File: README.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ We would like to thank [Keivan Beigi (@kayone)](https://github.com/kayone) for h
 ## Contributing
 
 This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+Contributor License Agreement (CLA) declaring that you have the right to, and do, grant us
```
Comment (Trenly) [inline README.md:115]: This is a legal statement. `actually` is needed here to properly convey intent

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by yao-msft — CHANGES_REQUESTED** [review]
Requesting changes until readme.md suggestions are applied (or reverted) and processor.csproj is updated

File: README.md (hunk 1 above)
Comment (skanda890) [reply README.md:10]: Then why did Grammarly suggest that?

**Comment by skanda890** [comment]
> https://github.com/microsoft/winget-cli/blob/91ee267ccd1bb6d9f976153e195572a5ee9746d8/src/Microsoft.Management.Configuration.Processor/Microsoft.Management.Configuration.Processor.csproj#L42

Done.

**Comment by skanda890** [comment]
Look at skanda890/winget-cli.

**Comment by skanda890** [comment]
Or look at https://github.com/skanda890/winget-cli/tree/snyk-fix-847d5fdee94697c54e714464cac3f5f8

**Review by skanda890 — COMMENTED** [review]
Fixed what you said.

**Review by skanda890 — COMMENTED** [review]
C

File: README.md [hunk 3]
```diff
@@ -1,11 +1,10 @@
-
 # ![WinGet Icon](.github/images/WindowsPackageManager_Assets/ICO/PNG/_40.png) Windows Package Manager
 
 ## WinGet Client
 
 ![winget install wingetcreate](.github/images/WingetInstall.gif)
 
-If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
+If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "Winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
```
Comment (denelon) [inline README.md:7]: This one should stay lower case since that's how it's presented in the client.

File: README.md [hunk 4]
```diff
@@ -45,11 +44,11 @@ Please read our [troubleshooting guide](/doc/troubleshooting/README.md).
 
 ## Administrator Considerations
 
-Installer behavior can be different depending on whether you are running **winget** with administrator privileges.
+Installer behavior can be different depending on whether you are running **Winget** with administrator privileges.
```
Comment (denelon) [inline README.md:47]: This one should be "Pascal" case. The product name when displayed as a single word should be "WinGet".

**Comment by denelon** [comment]
Sorry for the delay to provide feedback on my part. I've been traveling.

**Comment by skanda890** [comment]
@denelon, changes applied.

**Comment by yao-msft** [comment]
Just fyi, the security fixes for issues in https://github.com/skanda890/winget-cli/pull/3 are already applied with https://github.com/microsoft/winget-cli/pull/4083 and https://github.com/microsoft/winget-cli/pull/3495 and https://github.com/microsoft/winget-cli/pull/3355 so we should be good security wise.

**Pull request closed without merging**